common/static/common/js/common_libraries.js

```diff
@@ -1 +1 @@
-define(['domReady!', 'jquery', 'backbone', 'underscore', 'gettext']);
+define(['domReady!', 'jquery', 'jquery-migrate', 'backbone', 'underscore', 'gettext']);
```

Comment:
@dan-f I think this is a mistake, as 'jquery-migrate' is actually mapped to 'jquery' itself, right? I'm seeing errors in Studio trying to load jquery-migrate:

![image](https://cloud.githubusercontent.com/assets/5985072/13230911/5fb2d168-d975-11e5-9d66-95d5dafa02d7.png)